service/descriptors/ModuleDescriptor-template.json

```diff
@@ -29,6 +29,37 @@
         }
       ]
     },
+    {
+      "id": "workflow-nodes",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/nodes",
+          "permissionsRequired": ["workflow.nodes.collection.get"]
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/nodes",
+          "permissionsRequired": ["workflow.nodes.item.post"]
+        },
+        {
+          "methods": ["DELETE"],
+          "pathPattern": "/nodes/{id}",
+          "permissionsRequired": ["workflow.nodes.item.delete"]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/nodes/{id}",
+          "permissionsRequired": ["workflow.nodes.item.get"]
+        },
+        {
+          "methods": ["PUT"],
+          "pathPattern": "/nodes/{id}",
+          "permissionsRequired": ["workflow.nodes.item.put"]
+        }
+      ]
+    },
```

Comment:
What doe these new end points have to do with upgrading Kafka?

Is there a separate feature request that these belong to?
Is there a separate issue that needs bug fixing?

Are these Kafka end points?
If so, then why are we exposing Kafka this way through Okapi?
Is this how it is done?
Please document this.

Reply:
They have to do with functional testing of build, activate, and run of fw-cli. It is an incomplete backlog issue missing exposing the nodes API through Okapi.

Reply:
They are related to functionally testing a workflow and is a past technical debt that has not been resolved. There is no means to running a workflow from ui-workflow and we require fully functional testing of build, activate, run and trigger from event. All of which have been tested and provided in https://github.com/TAMULib/fw-cli/pull/96.

Reply:
Workflows can run fine without ui-workflow and have already been provent to do so.

This change should be a separate issue and not related to Kafka.
We have not yet planned out and groomed the functionality for running a workflow via ui-workflow.

Reply:
Why break fw-cli?

Reply:
This fixes fw-cli and changes nothing with ui-workflow.

Reply:
I am a bit confused on what the objective is to not support the initial client for the backend workflow modules.

Reply:
Going through Okapi is preferred and has been part of the original design of fw-cli. The changes to go through Okapi are minimal there and work as expected however requiring this API being exposed. It should be easy enough to remove when ui-workflow has feature parity and not require scripting per development sprint.